modules/Settings/Picklist/models/Module.php

```diff
@@ -372,4 +373,28 @@ public static function getInstanceFromModuleObject(vtlib\Module $moduleObj): sel
 		}
 		return $moduleModel;
 	}
+
+	/**
+	 * list of modules in which they appear picklist fields.
+	 *
+	 * @param array $pickListFields
+	 *
+	 * @return array
+	 */
+	public function listModuleInterdependentPickList(array $pickListFields): array
+	{
+		$interdependent = [];
+		$dataReader = (new App\Db\Query())->select(['tabid', 'fieldname'])
+			->from('vtiger_field')
+			->where(['fieldname' => $pickListFields])
+			->createCommand()->query();
+		while ($row = $dataReader->read()) {
+			$moduleName = \App\Module::getModuleName($row['tabid']);
+			$moduleModel = \Vtiger_Module_Model::getInstance($moduleName);
+			if ($moduleModel->isActive() && $moduleModel->getField($row['fieldname'])->isActiveField()) {
```

Comment:
```suggestion
			if ($moduleModel->isActive() && $moduleModel->getFieldByName($row['fieldname'])->isActiveField()) {
```
but add all conditions to query.